app/hub/Views/UpgradePlanView/UpgradePlanView.jsx

```diff
@@ -190,7 +184,7 @@ const UpgradePlanView = (props) => {
 
 	const premiumCTAButton = (position) => {
 		const utm_campaign = position === 'top' ? 'c_3' : 'c_4';
```

Comment:
Same deal here, could you add a set of parens around the ternary on the right side of the assignment?

Reply:
Done!